generate-release-notes.sh

```diff
@@ -16,7 +16,7 @@ Usage:
 
 Options:
 	--al2-gpu-nvidia-ver  (Optional) AL2 GPU NVIDIA version. If specified, then --al2-gpu-cuda-ver option is also required to be specified.
-	--al2-gpu-cuda-ver    (Optional) AL2 GPU CUDA version. If specified, then  --al2-gpu-nvidia optin is also required to be specified.
+	--al2-gpu-cuda-ver    (Optional) AL2 GPU CUDA version. If specified, then  --al2-gpu-nvidia option is also required to be specified.
```

Comment:
Q: should we also modify the check on L.77 to include `al2kernel5dot10gpu`?

Reply:
Thanks for calling out! Yes we should; L180-L181 should be modified as well. I will update this in the next revision.

Reply:
Thanks Dane!

Reply:
super nit & non-blocking. "--al2-gpu-nvidia" should be updated to "--al2-gpu-nvidia`-ver`" on line 19.

Reply:
Thanks @chienhanlin! Updated this in revision.